### DISCUSSION TITLE
Loaders and Options

### DISCUSSION THREAD

**Discussion by bdemick** [NationalSecurityAgency/ghidra#4536] [Q&A] [answered]
`analyzeHeadlessREADME.html` references the "most commonly used loaders and their arguments", but there are obviously a [ton of different loaders available](https://github.com/NationalSecurityAgency/ghidra/tree/ca60845ee1551f33d8e02b03503d67a1742d9251/Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion).

Does there exist any documentation on these loaders and their valid `-loader-option` strings and descriptions beyond the analyzeHeadless section on them? I've been trying to spot where I can enumerate these, but have come up empty following the parsing logic down from the `AnalyzeHeadless` implementation.

**Comment by ryanmkurtz** [comment, marked as answer]
No, if it's not in that headless README, then they are not documented anywhere (other than the source code itself).
